src/pygenesis/find.py

```diff
@@ -0,0 +1,269 @@
+"""Implements find endpoint to retrieve results based on query"""
+
+import pandas as pd
+
+from pygenesis.http_helper import get_response_from_endpoint
+
+pd.set_option("max_colwidth", None)
+pd.set_option("expand_frame_repr", False)
+
+
+class Results:
+    """
+    A class representing the result object of variables, statistics, cubes and tables.
+
+    Attributes:
+        df (pd.DataFrame): The DataFrame that contains the data.
+        category (str): Category of the result. E.g. "tables", "cubes".
+
+    Methods:
+        get_code(): Gets code based on the index of the object.
+        get_metadata(): Gets metadata based on the index of the object.
+    """
+
+    def __init__(self, result: pd.DataFrame, category: str):
+        """
+        Class that contains the results of a find query.
+
+        Args:
+            DataFrame (pd.DataFrame): Result of a search query.
+            category (str): Category of the result. E.g. "tables", "cubes"
+        """
+        self.df = result
+        self.category = category
+
+    def __repr__(self):
+        return self.df.to_markdown()
+
+    def __str__(self):
+        return self.df.to_markdown()
+
+    def __len__(self):
+        if len(self.df) > 0:
+            return len(self.df)
+        else:
+            return 0
+
+    def get_code(self, index: list):
```

Comment:
Hier fehlt der return Wert

Reply:
Ist doch 0, oder?

Reply:
Was meinst du mit 0? Die Funktion gibt doch eine Liste zurück oder? Genauer vermutlich List[str] weil Liste von Codenamen?

Reply:
Also ich meinte es fehlt in Zeile 47 der return value annotation für die Funktion `get_code() -> List[str?]`